### PR TITLE
Fix : fixed the broken links in CHANGELOG.MD

### PR DESCRIPTION
**Pull request by i-shivamsoni**
The project was started by @dblock, and a few years ago, we switched to the @oshi organization. #880 tried to remove all references to the old links, but missed the changelog!
So I Searched and replaced all links of the form https://github.com/dblock/oshi/pull to read https://github.com/oshi/oshi/pull.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -259,206 +260,206 @@ The default configuration file has been renamed to `oshi.properties` to prevent
...
+* Brought over lessons learned from [waffle](https://github.com/oshi/waffle) for building project from source.
+* [#50](https://github.com/oshi/oshi/pull/50): Added file store information - [@dbwiddis](https://github.com/dbwiddis).
+* [#51](https://github.com/oshi/oshi/pull/51): Added CPU Ticks and switched to OperatingSystemMXBean for CPU load / load average - [@dbwiddis](https://github.com/dbwiddis).
+* [#62](https://github.com/oshi/oshi/pull/62): Added Per-Processor CPU Load and Ticks - [@dbwiddis](https://github.com/dbwiddis).
 
 1.2 (6/13/2014)
 ================
 
 * Added TODO list and enhanced README documentation - [@ptitvert](https://github.com/ptitvert)
-* Added Travis-CI - [@dblock](https://github.com/dblock).
```
Comment (dbwiddis) [inline CHANGELOG.md:452]: Need to keep the references to committers' names/profiles

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 5.1.0 (in progress)
 ================
 * Your contribution here
+* [#1212](https://github.com/oshi/oshi/pull/1212): Fixed the broken links missed in  #880 in CHANGELOG.MD . - [@i-shivamsoni](https://github.com/i-shivamsoni).
```
Comment (dbwiddis) [inline CHANGELOG.md:4]: Probably not needed to add a changelog entry for this. :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -259,206 +260,206 @@ The default configuration file has been renamed to `oshi.properties` to prevent
...
+  [#72](https://github.com/oshi/oshi/pull/72): Added support for Windows 10 & Windows Server 2016 - [@laurent-r](https://github.com/laurent-r).
+* [#75](https://github.com/oshi/oshi/pull/75): Added uptime information - [@dbwiddis](https://github.com/dbwiddis).
+* [#76](https://github.com/oshi/oshi/pull/76): Better linux CPU processor counting - [@dbwiddis](https://github.com/dbwiddis).
+* [#78](https://github.com/oshi/oshi/pull/78): Execute FileSystemView on Swing's Event Dispatch Thread - [@dbwiddis](https://github.com/dbwiddis).
 
 1.3 (6/27/2015)
 ================
 * Upgraded to Java 7 base support
 * Upgraded JNA to 4.1.0
-* Brought over lessons learned from [waffle](https://github.com/dblock/waffle) for building project from source.
```
Comment (dbwiddis) [inline CHANGELOG.md:443]: I think waffle has a new URL as well: https://github.com/Waffle/waffle

File: CHANGELOG.md [hunk 4]
```diff
@@ -468,4 +469,4 @@ The default configuration file has been renamed to `oshi.properties` to prevent
 1.0 (6/23/2010)
 ===============
 
-* Initial public release - [@dblock](https://github.com/dblock).
+* Initial public release - [@oshi](https://github.com/oshi).
```
Comment (dbwiddis) [inline CHANGELOG.md:472]: Again, keep original committer names.

**Comment by i-shivamsoni** [comment]
I have done the Changes which are requested.

**Comment by i-shivamsoni** [comment]
> Not quite so simple as just replacing the one string.

Sorry for the errors, kindly review the request now I have fixed the errors as per requested changes.

File: CHANGELOG.md [hunk 5]
```diff
@@ -259,206 +259,206 @@ The default configuration file has been renamed to `oshi.properties` to prevent
...
-* [#13](https://github.com/dblock/oshi/pull/13): Support for Windows 8.1 and Windows Server 2008 R2 - [@NagyGa1](https://github.com/NagyGa1).
-* [#15](https://github.com/dblock/oshi/pull/15), [#18](https://github.com/dblock/oshi/pull/18): Added support for CPU load - [@kamenitxan](https://github.com/kamenitxan), [@Sorceror](https://github.com/Sorceror).
-* [#25](https://github.com/dblock/oshi/pull/25), [#29](https://github.com/dblock/oshi/pull/29): Included inactive/reclaimable memory amount in GlobalMemory#getAvailable on Mac/Linux - [@dbwiddis](https://github.com/dbwiddis).
-* [#27](https://github.com/dblock/oshi/pull/27): Replaced all Mac OS X command line parsing with JNA or System properties - [@dbwiddis](https://github.com/dbwiddis).
-* [#30](https://github.com/dblock/oshi/pull/30): Added processor vendor frequency information - [@alessiofachechi](https://github.com/alessiofachechi).
-* [#32](https://github.com/dblock/oshi/pull/32): Added battery state information - [@dbwiddis](https://github.com/dbwiddis).
+* Added Travis-CI - Added Travis-CI - [@dblock](https://github.com/dblock).
```
Comment (dbwiddis) [inline CHANGELOG.md:452]: One last fix.  Looks like the "Added Travis-CI" and "Initial Public Release" strings below got duplicated.

Comment (i-shivamsoni) [reply CHANGELOG.md]: It's done!!. Thanks for the patience.

File: CHANGELOG.md [hunk 6]
```diff
@@ -259,206 +260,206 @@ The default configuration file has been renamed to `oshi.properties` to prevent
...
-* [#7](https://github.com/dblock/oshi/pull/7): Added Mac OS X Support - [@ptitvert](https://github.com/ptitvert).
-* [#13](https://github.com/dblock/oshi/pull/13): Support for Windows 8.1 and Windows Server 2008 R2 - [@NagyGa1](https://github.com/NagyGa1).
-* [#15](https://github.com/dblock/oshi/pull/15), [#18](https://github.com/dblock/oshi/pull/18): Added support for CPU load - [@kamenitxan](https://github.com/kamenitxan), [@Sorceror](https://github.com/Sorceror).
-* [#25](https://github.com/dblock/oshi/pull/25), [#29](https://github.com/dblock/oshi/pull/29): Included inactive/reclaimable memory amount in GlobalMemory#getAvailable on Mac/Linux - [@dbwiddis](https://github.com/dbwiddis).
-* [#27](https://github.com/dblock/oshi/pull/27): Replaced all Mac OS X command line parsing with JNA or System properties - [@dbwiddis](https://github.com/dbwiddis).
-* [#30](https://github.com/dblock/oshi/pull/30): Added processor vendor frequency information - [@alessiofachechi](https://github.com/alessiofachechi).
-* [#32](https://github.com/dblock/oshi/pull/32): Added battery state information - [@dbwiddis](https://github.com/dbwiddis).
+* Added Travis-CI - [@oshi](https://github.com/oshi).
```
Comment (dbwiddis) [inline CHANGELOG.md]: This still needs to be the original `@dblock`.

File: CHANGELOG.md (hunk 4 above)
Comment (dbwiddis) [inline CHANGELOG.md]: This here also needs to be `@dblock`

File: CHANGELOG.md (hunk 6 above)
Comment (i-shivamsoni) [reply CHANGELOG.md]: Annotation 2020-05-13 002225
Please refer to this commit 4b01753. I have already fixed it.

Comment (dbwiddis) [reply CHANGELOG.md]: Ah, somehow I was looking at a non-refreshed version. Looks great!  Merging.

**Pull request merged**